python-client/giskard/scanner/result.py

```diff
@@ -32,7 +32,7 @@ def _repr_html_(self):
 
     def to_html(self, filename=None, embed=False):
         env = Environment(
-            loader=PackageLoader("giskard.scanner", "templates"),
+            loader=PackageLoader("giskard", "templates"),
```

Comment:
We should definitely reorganize this into a visualization package, e.g. `giskard.visualization` or similar where we move both the templates and the logic related to them.